library/Notifications/Widget/ItemList/ScheduleList.php

```diff
@@ -14,4 +15,10 @@ protected function getItemClass(): string
     {
         return ScheduleListItem::class;
     }
+
+    protected function init(): void
+    {
+        $this->getAttributes()
+            ->set('data-icinga-detail-url', (string) Url::fromPath('notifications/schedule'));
```

Comment:
Same change as for `ContactGroupList`.
Remember to introduce null values to `Links::schedule()` as well.